x/mint/types/params.go

```diff
@@ -219,19 +219,38 @@ func validateDistributionProportions(i interface{}) error {
 	return nil
 }
 
-func validateDeveloperRewardsReceiver(i interface{}) error {
-	v, ok := i.(string)
+func validateWeightedDeveloperRewardsReceivers(i interface{}) error {
+	v, ok := i.([]WeightedAddress)
 	if !ok {
 		return fmt.Errorf("invalid parameter type: %T", i)
 	}
 
 	// fund community pool when rewards address is empty
-	if v == "" {
+	if len(v) == 0 {
 		return nil
 	}
 
-	_, err := sdk.AccAddressFromBech32(v)
-	return err
+	weightSum := sdk.NewDec(0)
+	for i, w := range v {
+		// TODO: should we allow portion to be "" to go to community pool?
```

Comment:
Yeah I suppose that's probably a good idea.

Reply:
Added